op-challenger/challenger.go

```diff
@@ -40,11 +41,26 @@ func Main(logger log.Logger, cfg *config.Config) error {
 
 	agent := fault.NewAgent(loader, gameDepth, trace, responder, cfg.AgreeWithProposedOutput, logger)
 
+	caller, err := fault.NewFaultCaller(cfg.GameAddress, client)
+	if err != nil {
+		return fmt.Errorf("failed to bind the fault contract: %w", err)
+	}
+
 	logger.Info("Fault game started")
 
 	for {
 		logger.Info("Performing action")
 		_ = agent.Act()
+		status, err := caller.GetGameStatus(context.Background())
```

Comment:
This is a good thing to pull out into a function which either returns the info or does the logging.

Reply:
Ah I like that design, will do.